.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Documentation
+about: Request an improvement to the existing documentation.
+title: ''
+labels: Documentation
+assignees: ''
+---
+
+
+#### Proposed documentation ehancement
+*Describe your proposed enhancement in detail. If you are requesting a new
+glossary term, please include a proposed definition.*
```

Comment:
What about making this part a comment?

```suggestion
<!--Describe your proposed enhancement in detail. If you are requesting a new
glossary term, please include a proposed definition.-->
```

Reply:
Sorry, but what is a "glossary term" in this context ? The only glossary I can find in the docs is [here](https://nilearn.github.io/introduction.html#glossary-machine-learning-vocabulary).

Reply:
The oldest open issue is #75 and @NicolasGensollen self-assigned it a month ago, so I think creating a glossary is on the to-do list.

Reply:
I might remove that clarification until the new glossary is set-up, then, only since I found it a bit confusing. But not a strong opinion !

Reply:
Indeedn this may not be something to put forward in the template.

Reply:
@tsalo @emdupre I was indeed advertising for an upcoming glossary that I'm planning to do at some point. But you're right, this is confusing for now. I removed this comment and we'll add it later if necessary.